app/views/articles/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Showing article details</h1>
+
+<p><strong>Title: </strong><%= @article.title %></p>
```

Comment:
`<% %>` embeds the ruby code while `=` indicates rendering the ruby code in the `<% %>`